packages/Modal/src/modal.scss

```diff
@@ -300,12 +305,23 @@
         }
     }
 
+    .ids-modal__carousel {
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
```

Comment:
I feel like we should always set all three, whether you separate the props or not.
I like to do flex: 1 1 auto; to ensure each browser uses the same.